src/main/java/com/scriptbasic/executors/operators/AbstractBinaryFullCircuitOperator.java

```diff
@@ -15,14 +15,22 @@
  */
 public abstract class AbstractBinaryFullCircuitOperator extends
         AbstractBinaryOperator {
+	
+	protected RightValue getLeftEvaluatedOperand(final Interpreter interpreter) throws ScriptBasicException {
```

Comment:
Tabulation is messed up by one space. Please pay attention. Such small things lead to a maintenance nightmare in the long run.